common/display/src/main/scala/uk/ac/wellcome/display/models/DisplayImage.scala

```diff
@@ -30,7 +34,12 @@ object DisplayImage {
     new DisplayImage(
       id = image.id.canonicalId,
       locations = Seq(DisplayDigitalLocation(image.location)),
-      source = DisplayImageSource(image.source)
+      source = DisplayImageSource(image.source),
+      visuallySimilar = None
     )
 
+  def apply(image: AugmentedImage, similar: Seq[AugmentedImage]): DisplayImage =
+    DisplayImage(image).copy(
+      visuallySimilar = Some(similar.map(DisplayImage.apply))
+    )
```

Comment:
What is the advantage of having a separate `apply` for this? Could we not pass this an as an optional argument to the other `apply`?

Also might be worth having some tests that things are included correctly, like in the `DisplayWork`

Reply:
Good point re the apply.

Yeah, I will add some tests, but as a general point I'm not very convinced of the value of the display model tests. They effectively get re-tested in the API tests, and we don't add them consistently so they're not especially useful as documentation either...

Reply:
Actually, the apply doesn't work. Because the case class has default values, and defining an `apply` in the companion object is an overload, it doesn't compile if that `apply` has default arguments: `multiple overloaded alternatives of method apply define default arguments`.